collections/without_all.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+/** Used as the size to enable large array optimizations. */
+const LARGE_ARRAY_SIZE = 200;
+
+/**
+ * Returns an array excluding all given values.
+ *
+ * Example:
+ *
+ * ```ts
+ * import { withoutAll } from "./withoutAll.ts";
+ *
+ * const withoutList = withoutAll([2, 1, 2, 3], [1, 2]);
+ *
+ * ```
+ */
+export function withoutAll<T>(
+  array: Array<T>,
+  values: Array<T>,
+): Array<T> {
```

Comment:
From the function signature and the description, it is unclear what this function actually does. Is this meant to always return `[]` since "an array excluding all given values" is returned?

Couple suggestions:
1. Tell the user what `withoutList` looks like after the function call with `assertEquals` or a comment.
2. Give parameters better names. `array` and `values` does not provide additional information to the user.
3. Maybe a better description would also be helpful.

Reply:
Sure, You can read `Collections/README.md`, it contains clear description with example.